File: sagrada-familia-web/src/app/shared/interfaces/responses/alimento.response.ts
export interface AlimentoResponse {
    id: number;
    categoriaId: number;
    categoriaNombre: string;
    nombre: string;
    descripcion: string | null;
    edadMinimaIntro: number;
    edadMaxima: number | null;
    recomendacion: string | null;
}

export interface CategoriaResponse {
    id: number;
    nombre: string;
    descripcion: string | null;
    totalAlimentos: number;
}

export interface CategoriaPadreUI extends CategoriaResponse {
    clase: string;
    icono: string;
}

File: SagradaFamilia/SagradaFamilia.Application/DTOs/Alimentos/AlimentoResponse.cs
namespace SagradaFamilia.Application.DTOs.Alimentos
{
    public class AlimentoResponse
    {
        public int Id { get; set; }
        public int CategoriaId { get; set; }
        public string CategoriaNombre { get; set; } = string.Empty;
        public string Nombre { get; set; } = string.Empty;
        public string? Descripcion { get; set; }
        public int EdadMinimaIntro { get; set; }
        public int? EdadMaxima { get; set; }
        public string? Recomendacion { get; set; }
    }
}
